### DISCUSSION TITLE
Does Ink not support some commonly used markdown syntax, such as blockquote and table? And, is there a plan to support mathematical formulas?

### DISCUSSION THREAD

**Discussion by zxC0der** [davidmyersdev/octo#161] [Q&A]
I'm looking for an markdown editor rendering in-place, Ink seems to be a good choice

**Comment by davidmyersdev** [comment]
Hello @zxC0der!

Ink does support those syntaxes, but it does not apply any special styles to them _yet_. I've been considering a good approach for tables, formulas, and mermaid diagrams. I will likely need to use block widgets (similar to how images are displayed above the markdown line) to show the rendered versions of the formulas or diagrams, but I think we could do something better for tables.

I'd love to hear any thoughts you have!
